simpleimage/make_rootfs.sh

```diff
@@ -214,7 +242,21 @@ add_asound_state() {
 
 # Run stuff in new system.
 case $DISTRO in
+    centos)
+		mv "$DEST/etc/resolv.conf" "$DEST/etc/resolv.conf.dist"
+        cp /etc/resolv.conf "$DEST/etc/resolv.conf"
+        add_platform_scripts 
+        add_mackeeper_service
+        add_corekeeper_service
+        add_ssh_keygen_service
+        add_disp_udev_rules
+        add_wifi_module_autoload
+        add_asound_state
+		rm -f "$DEST/etc/resolv.conf"
+		mv "$DEST/etc/resolv.conf.dist" "$DEST/etc/resolv.conf"
+        ;;
 	arch)
+        do_prepare_chroot
```

Comment:
There seems to be indentation issues, please use the existing indentation and style consistently.